pydash/pydash_app/dashboard/__init__.py

```diff
@@ -36,3 +42,35 @@ def dashboards_of_user(user_id):
     :return: A list of Dashboard-entities.
     """
     return [db for db in repository.all() if db.user_id == user_id]
+
+
+def find_verified_dashboard(dashboard_id):
+    """
+    Verifies if a given dashboard_id is correct and if the current user has access
+    to the dashboard.
+    :param dashboard_id: The UUID of the dashboard to be validated.
+    :return: True if the dashboard is valid, else False followed by the result and
+    the http error code.
+    """
+
+    try:
+        dashboard = pydash_app.dashboard.find(dashboard_id)
+    except KeyError:
+        result = {'message': f'Dashboard_id {dashboard_id} is not found.'}
+        logger.warning(f'Endpoint_boxplots failure - Dashboard_id {dashboard_id} is not found. '
```

Comment:
This error message does not belong here, it will be wrong if `find_verified_dashboard` is called in a different place than the endpoint boxplots route